### ISSUE TITLE
Ignores whitelist entry

### ISSUE THREAD

**Issue by Marenz** [codespell-project/codespell#1128]
We just got auto-updated to the latest version (1.15) and now it wants to correct:
```
errorString ===> error string
```

I added the term in my whitelist file, but it ignores that entry and even suggests to correct it in the whitelist file itself.

Example of this (see the "Check Spelling" part): https://circleci.com/gh/ethereum/solidity/128016?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

**Comment by Marenz** [comment]
Okay, I guess I needed to add `errorstring` instead of `errorString`. So the matching is done case insensitive but the whitelist matching is done case sensitive? 

I am not sure about a good solution, but I feel that "errorString" shouldn't exactly match "errorstring", I mean, especially with the camel case?

Anyway, feel free to close this if you don't see an issue here.

**Comment by Marenz** [comment]
I suppose a possible solution can be that "camelCase" could be interpreted as "camel case" for the matching.. I see there is a similar ticket https://github.com/codespell-project/codespell/issues/196 about this issue already.

**Comment by luzpaz** [comment]
yea, I get a lot of false positives with this entry as well from `errorstring->error string`. Beside the standing issue of #196 i think we should remove this one from the dictionary.

**Comment by peternewman** [comment]
This got removed from the dictionary, but I'd agree it feels like you've found a bug if the whitelist and dictionary use different types of case matching.

You do have to blacklist your whitelist file from checking, again perhaps that's something that should happen automagically.

**Comment by tobylane** [comment]
> $ echo currentY | codespell -L currentY -
1: currentY
	currentY ==> currently

Is this the same bug? I installed from git.

**Comment by peternewman** [comment]
> > $ echo currentY | codespell -L currentY -
> > 1: currentY
> > currentY ==> currently
> 
> Is this the same bug? I installed from git.

Yep @tobylane , the -L and -I flags are case sensitive as per the dictionary, so `grep -i` the data folder to confirm how the typo appears in the dictionary, but in your case this should work:
`echo currentY | codespell -L currenty -`
